build.assets/Makefile

```diff
@@ -335,13 +336,13 @@ $(GOCACHE):
 .PHONY: $(TARGETS)
 $(TARGETS): | $(GOCACHE)
 	@echo -e "\n----> Building $@ binary...\n"
-	go build -o $@ -ldflags $(GRAVITY_LINKFLAGS) $(GRAVITY_BUILDFLAGS) -tags "$(GRAVITY_BUILDTAGS)" $(GRAVITY_PKG_PATH)/tool/$(@F)
+	GO111MODULE=on GOMOD=$(SRCDIR)/go.mod go build -mod=vendor -o $@ -ldflags $(GRAVITY_LINKFLAGS) $(GRAVITY_BUILDFLAGS) -tags "$(GRAVITY_BUILDTAGS)" $(GRAVITY_PKG_PATH)/tool/$(@F)
```

Comment:
Hmm, GOMOD is one of the environment variables that's being synthesized but not read:
```
$ go help environment
...
Additional information available from 'go env' but not read from the environment:
	...
	GOMOD
		The absolute path to the go.mod of the main module,
		or the empty string if not using modules.
```

Reply:
I am not sure I followed what did you mean. BTW, when I deleted GOMOD from the line a build is still working, but I believe in my previous PR it was not working. :man_shrugging:

Reply:
I mean go tools do not read from `GOMOD` but simply populate it to show which go.mod file is in use.